scipy/stats/_stats_py.py

```diff
@@ -7525,37 +7550,40 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
                                      alternative=alternative)
         df, denom, estimate = NaN, NaN, NaN
 
+        # _axis_nan_policy decorator doesn't play well with strings
```

Comment:
Suggestion: review with "Hide whitespace". I added an early return here to dedent the important part of the function.